api/gateway/stripe/stripe.cfc

```diff
@@ -217,18 +348,25 @@
 
 		<!--- add authentication --->		
 		<cfset headers["authorization"] = "Bearer #getSecretKey()#" />
+		<!--- if we want to override the stripe API version, we can set it in the config with "ApiVersion".  Using 'latest' overrides to current version --->
+		<cfif len(getApiVersion())>
+			<!--- https://groups.google.com/a/lists.stripe.com/forum/#!topic/api-discuss/V4sYRlHwalc --->
+			<cfset headers["Stripe-Version"] = getApiVersion() />
+		</cfif>
+		<!--- help track where this request was made from --->
+		<cfset headers["User-Agent"] = "Stripe/v1 cfpayment/#variables.cfpayment.GATEWAY_VERSION#" />
 
 		<!--- process standard and common CFPAYMENT mappings into Braintree-specific values --->
 		<cfif structKeyExists(arguments.options, "description")>
 			<cfset p["description"] = arguments.options.description />
 		</cfif>
 		<cfif structKeyExists(arguments.options, "tokenId")>
-			<cfset p["customer"] = arguments.options.tokenId />
+			<cfset p["card"] = arguments.options.tokenId />
```

Comment:
Can you revert this line back to p["customer"] until I resolve https://github.com/ghidinelli/cfpayment/pull/6 ?